Src/FluentAssertions/StringContainmentConstraints.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Diagnostics;
+using FluentAssertions.Primitives;
+
+namespace FluentAssertions
+{
+    [DebuggerNonUserCode]
+    public class StringContainmentConstraints : AndConstraint<StringAssertions>
+    {
+        private readonly string expected;
+        private readonly int occurrences;
+        private readonly StringAssertions parentConstraint;
+
+        public StringContainmentConstraints(StringAssertions parentConstraint, string expected) : base(parentConstraint)
+        {
+            this.parentConstraint = parentConstraint;
+            this.expected = expected;
+            occurrences = CountOccurrences(parentConstraint.Subject, expected, StringComparison.Ordinal);
+        }
+
+        private static int CountOccurrences(string actual, string expected, StringComparison comparison)
```

Comment:
🔧 How about inlining comparison as it's always StringComparison.Ordinal?

Reply:
Sure, I forgot to remove this parameter.
I will add this when I will implement similar API for `ContainEquivalentOf`.